README.md

```diff
@@ -8,8 +8,8 @@ The tricky part is that upper-case bitcoin might mean something different than l
 
 | Term | Definition |
 | --- | --- |
-| Bitcoin | |
-| bitcoin | |
+| Bitcoin | The Bitcoin blockchain |
```

Comment:
I think "Bitcoin" is more than *just* the blockchain.  It's also the system of nodes that maintains it, the underlying protocol, and the entire design.

Reply:
In a broader sense, the miners and core devs too are a part of the ecosystem, but I don't think we intend them when we talk about Bitcoin. 
Bitcoin is a cumulative of the network, the blockchain and the protocol consensus.
We could call Bitcoin the world's first decentralized blockchain system of payment, what say ?
[Wikipedia](https://en.wikipedia.org/wiki/Bitcoin) differentiates between 'Bitcoin' and 'bitcoin' but Oxford recommends the use of just lowercase 'bitcoin'.